simpleclient_hotspot/src/main/java/io/prometheus/client/hotspot/ThreadExports.java

```diff
@@ -61,8 +61,24 @@ void addThreadMetrics(List<MetricFamilySamples> sampleFamilies) {
           "jvm_threads_started_total",
           "Started thread count of a JVM",
           threadBean.getTotalStartedThreadCount()));
+
+
+    sampleFamilies.add(
+        new GaugeMetricFamily(
+        "jvm_threads_deadlocked",
+        "Deadlocked thread count of a JVM",
+        nullSafeArrayLength(threadBean.findDeadlockedThreads())));
+
+    sampleFamilies.add(
+        new GaugeMetricFamily(
+        "jvm_threads_monitor_deadlocked",
```

Comment:
deadlocked_monitor would be more typical.

I'd suggest expanding the help a bit too so users will get idea of the difference between these two metrics.